src/cml.js

```diff
@@ -69,49 +70,15 @@ const getDriver = (opts) => {
 
   throw new Error(`driver ${driver} unknown!`);
 };
-
-const fixGitSafeDirectory = () => {
```

Comment:
This also solves the permission issues for any `git` commands ran by users. Should it be ported to the new library?

Reply:
Im testing it. ideally should not be the case. We do not depend on git.